api/controllers/user.go

```diff
@@ -109,6 +109,17 @@ func (uc *UserController) Login(w http.ResponseWriter, r *http.Request) {
 	w.WriteHeader(201)
 }
 
+// @Summary Update user information
+// @Description Updates the details of an existing user based on the provided user ID and payload
+// @Tags User
+// @Produce json
+// @Param userId path string true "User ID"
+// @Param user body dto.UserUpdateDto true "User update payload"
+// @Success 200 {object} dto.UserUpdateDto
+// @Failure 400
+// @Failure 404
+// @Failure 500
+// @Router /users/{userId} [put]
```

Comment:
aqui é `/user` o método á `patch` e altera o parametro pra `userID`

```suggestion
// @Router /user/{userID} [path]
```

Reply:
userID